tests/psalm-baseline.xml

```diff
@@ -125,4 +125,9 @@
       <code><![CDATA[$result && count($result) > 0]]></code>
     </RedundantCondition>
   </file>
+  <file src="lib/WOPI/SettingsUrl.php">
+    <RedundantCondition>
+      <code><![CDATA[$result && count($result) > 0]]></code>
+    </RedundantCondition>
+  </file>
```

Comment:
This condition does not seem to be there anymore in the code, so can also be dropped from the baseline

Reply:
Done